packages/ui/src/views/repo/repo-summary/repo-summary.tsx

```diff
@@ -181,17 +181,17 @@ export function RepoSummaryView({
             */}
             <ListActions.Root>
               <ListActions.Left>
-                <ButtonGroup className="gap-2.5">
+                <ButtonLayout>
                   {branchSelectorRenderer}
                   <SearchFiles
                     navigateToFile={navigateToFile}
                     filesList={filesList}
                     useTranslationStore={useTranslationStore}
                   />
-                </ButtonGroup>
+                </ButtonLayout>
               </ListActions.Left>
               <ListActions.Right>
-                <ButtonGroup className="gap-2.5">
+                <ButtonLayout className="gap-2.5">
```

Comment:
Sorry, I meant to ask if we needed the custom gap here too?
```suggestion
                <ButtonLayout>
```